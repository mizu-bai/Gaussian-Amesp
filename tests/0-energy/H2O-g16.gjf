%nproc=2
%chk=H2O-g16.chk
#p b3lyp em=gd3bj 6-31G** nosymm

H2O energy task (g16)

0 1
 O                  0.00000000   -0.00000000   -0.11085125
 H                  0.00000000   -0.78383672    0.44340501
 H                 -0.00000000    0.78383672    0.44340501

