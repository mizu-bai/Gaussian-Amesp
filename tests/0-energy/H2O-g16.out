 Entering Gaussian System, Link 0=g16
 Initial command:
 /opt/g16/l1.exe "/home/jh-li/scratch/g16/Gau-353574.inp" -scrdir="/home/jh-li/scratch/g16/"
 Entering Link 1 = /opt/g16/l1.exe PID=    353579.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                12-Oct-2023 
 ******************************************
 %nproc=2
 Will use up to    2 processors via shared memory.
 %chk=H2O-g16.chk
 --------------------------------
 #p b3lyp em=gd3bj 6-31G** nosymm
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,15=1,17=6,18=5,40=1/2;
 3/5=1,6=6,7=101,11=2,25=1,30=1,74=-5,124=41/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,9=1/99;
 Leave Link    1 at Thu Oct 12 10:24:07 2023, MaxMem=           0 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l101.exe)
 ---------------------
 H2O energy task (g16)
 ---------------------
 Symbolic Z-matrix:
 Charge =  0 Multiplicity = 1
 O                     0.        0.       -0.11085 
 H                     0.       -0.78384   0.44341 
 H                     0.        0.78384   0.44341 
 
 ITRead=  0  0  0
 MicOpt= -1 -1 -1
 NAtoms=      3 NQM=        3 NQMF=       0 NMMI=      0 NMMIF=      0
                NMic=       0 NMicF=      0.
                    Isotopes and Nuclear Properties:
 (Nuclear quadrupole moments (NQMom) in fm**2, nuclear magnetic moments (NMagM)
  in nuclear magnetons)

  Atom         1           2           3
 IAtWgt=          16           1           1
 AtmWgt=  15.9949146   1.0078250   1.0078250
 NucSpn=           0           1           1
 AtZEff=  -0.0000000  -0.0000000  -0.0000000
 NQMom=    0.0000000   0.0000000   0.0000000
 NMagM=    0.0000000   2.7928460   2.7928460
 AtZNuc=   8.0000000   1.0000000   1.0000000
 Leave Link  101 at Thu Oct 12 10:24:07 2023, MaxMem=   104857600 cpu:               0.2 elap:               0.1
 (Enter /opt/g16/l202.exe)
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0        0.000000    0.000000   -0.110851
      2          1           0        0.000000   -0.783837    0.443405
      3          1           0        0.000000    0.783837    0.443405
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3
     1  O    0.000000
     2  H    0.960000   0.000000
     3  H    0.960000   1.567673   0.000000
 Symmetry turned off by external request.
 Stoichiometry    H2O
 Framework group  C2V[C2(O),SGV(H2)]
 Deg. of freedom     2
 Full point group                 C2V     NOp   4
 Rotational constants (GHZ):         919.0227249         408.0852063         282.5991538
 Leave Link  202 at Thu Oct 12 10:24:07 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l301.exe)
 Standard basis: 6-31G(d,p) (6D, 7F)
 Ernie: Thresh=  0.10000D-02 Tol=  0.10000D-05 Strict=F.
    25 basis functions,    42 primitive gaussians,    25 cartesian basis functions
     5 alpha electrons        5 beta electrons
       nuclear repulsion energy         9.1571759236 Hartrees.
 IExCor=  402 DFT=T Ex+Corr=B3LYP ExCW=0 ScaHFX=  0.200000
 ScaDFX=  0.800000  0.720000  1.000000  0.810000 ScalE2=  1.000000  1.000000
 IRadAn=      5 IRanWt=     -1 IRanGd=            0 ICorTp=0 IEmpDi=141
 NAtoms=    3 NActive=    3 NUniq=    3 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned off.
 R6Disp:  Grimme-D3(BJ) Dispersion energy=       -0.0005735821 Hartrees.
 Nuclear repulsion after empirical dispersion term =        9.1566023416 Hartrees.
 Leave Link  301 at Thu Oct 12 10:24:07 2023, MaxMem=   104857600 cpu:               0.1 elap:               0.0
 (Enter /opt/g16/l302.exe)
 NPDir=0 NMtPBC=     1 NCelOv=     1 NCel=       1 NClECP=     1 NCelD=      1
         NCelK=      1 NCelE2=     1 NClLst=     1 CellRange=     0.0.
 One-electron integrals computed using PRISM.
 NBasis=    25 RedAO= T EigKep=  2.21D-02  NBF=    25
 NBsUse=    25 1.00D-06 EigRej= -1.00D+00 NBFU=    25
 Precomputing XC quadrature grid using
 IXCGrd= 4 IRadAn=           5 IRanWt=          -1 IRanGd=           0 AccXCQ= 0.00D+00.
 Generated NRdTot=       0 NPtTot=           0 NUsed=           0 NTot=          32
 NSgBfM=    25    25    25    25    25 MxSgAt=     3 MxSgA2=     3.
 Leave Link  302 at Thu Oct 12 10:24:07 2023, MaxMem=   104857600 cpu:               0.2 elap:               0.1
 (Enter /opt/g16/l303.exe)
 DipDrv:  MaxL=1.
 Leave Link  303 at Thu Oct 12 10:24:08 2023, MaxMem=   104857600 cpu:               0.0 elap:               0.0
 (Enter /opt/g16/l401.exe)
 ExpMin= 1.61D-01 ExpMax= 5.48D+03 ExpMxC= 8.25D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Symmetry not used in FoFCou.
 Harris En= -76.4590388976811    
 JPrj=0 DoOrth=F DoCkMO=F.
 Leave Link  401 at Thu Oct 12 10:24:08 2023, MaxMem=   104857600 cpu:               0.5 elap:               0.3
 (Enter /opt/g16/l502.exe)
 Keep R1 ints in memory in canonical form, NReq=1779930.
 FoFCou: FMM=F IPFlag=           0 FMFlag=           0 FMFlg1=           0
         NFxFlg=           0 DoJE=F BraDBF=F KetDBF=F FulRan=T
         wScrn=  0.000000 ICntrl=       600 IOpCl=  0 I1Cent=           0 NGrid=           0
         NMat0=    1 NMatS0=    325 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Symmetry not used in FoFCou.
 Two-electron integral symmetry not used.
 Closed shell SCF:
 Using DIIS extrapolation, IDIIS=  1040.
 NGot=   104857600 LenX=   104802784 LenY=   104801718
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 Integral accuracy reduced to 1.0D-05 until final iterations.

 Cycle   1  Pass 0  IDiag  1:
 E= -76.3407885352877    
 DIIS: error= 6.47D-02 at cycle   1 NSaved=   1.
 NSaved= 1 IEnMin= 1 EnMin= -76.3407885352877     IErMin= 1 ErrMin= 6.47D-02
 ErrMax= 6.47D-02  0.00D+00 EMaxC= 1.00D-01 BMatC= 1.07D-01 BMatP= 1.07D-01
 IDIUse=3 WtCom= 3.53D-01 WtEn= 6.47D-01
 Coeff-Com:  0.100D+01
 Coeff-En:   0.100D+01
 Coeff:      0.100D+01
 Gap=     0.214 Goal=   None    Shift=    0.000
 GapD=    0.214 DampG=1.000 DampE=0.500 DampFc=0.5000 IDamp=-1.
 Damping current iteration by 5.00D-01
 RMSDP=2.21D-02 MaxDP=2.19D-01              OVMax= 1.98D-01

 Cycle   2  Pass 0  IDiag  1:
 E= -76.3681381453908     Delta-E=       -0.027349610103 Rises=F Damp=T
 DIIS: error= 1.27D-02 at cycle   2 NSaved=   2.
 NSaved= 2 IEnMin= 2 EnMin= -76.3681381453908     IErMin= 2 ErrMin= 1.27D-02
 ErrMax= 1.27D-02  0.00D+00 EMaxC= 1.00D-01 BMatC= 6.17D-03 BMatP= 1.07D-01
 IDIUse=3 WtCom= 8.73D-01 WtEn= 1.27D-01
 Coeff-Com:  0.179D+00 0.821D+00
 Coeff-En:   0.294D+00 0.706D+00
 Coeff:      0.193D+00 0.807D+00
 Gap=     0.345 Goal=   None    Shift=    0.000
 RMSDP=2.68D-03 MaxDP=3.69D-02 DE=-2.73D-02 OVMax= 1.14D-01

 Cycle   3  Pass 0  IDiag  1:
 E= -76.4194439862674     Delta-E=       -0.051305840877 Rises=F Damp=F
 DIIS: error= 1.60D-03 at cycle   3 NSaved=   3.
 NSaved= 3 IEnMin= 3 EnMin= -76.4194439862674     IErMin= 3 ErrMin= 1.60D-03
 ErrMax= 1.60D-03  0.00D+00 EMaxC= 1.00D-01 BMatC= 9.91D-05 BMatP= 6.17D-03
 IDIUse=3 WtCom= 9.84D-01 WtEn= 1.60D-02
 Coeff-Com:  0.269D-01-0.174D-03 0.973D+00
 Coeff-En:   0.000D+00 0.000D+00 0.100D+01
 Coeff:      0.264D-01-0.172D-03 0.974D+00
 Gap=     0.357 Goal=   None    Shift=    0.000
 RMSDP=2.37D-04 MaxDP=2.55D-03 DE=-5.13D-02 OVMax= 2.16D-03

 Cycle   4  Pass 0  IDiag  1:
 E= -76.4195086533013     Delta-E=       -0.000064667034 Rises=F Damp=F
 DIIS: error= 3.12D-04 at cycle   4 NSaved=   4.
 NSaved= 4 IEnMin= 4 EnMin= -76.4195086533013     IErMin= 4 ErrMin= 3.12D-04
 ErrMax= 3.12D-04  0.00D+00 EMaxC= 1.00D-01 BMatC= 2.55D-06 BMatP= 9.91D-05
 IDIUse=3 WtCom= 9.97D-01 WtEn= 3.12D-03
 Coeff-Com:  0.476D-02-0.154D-01 0.223D+00 0.787D+00
 Coeff-En:   0.000D+00 0.000D+00 0.000D+00 0.100D+01
 Coeff:      0.474D-02-0.154D-01 0.223D+00 0.788D+00
 Gap=     0.357 Goal=   None    Shift=    0.000
 RMSDP=2.98D-05 MaxDP=2.48D-04 DE=-6.47D-05 OVMax= 3.20D-04

 Cycle   5  Pass 0  IDiag  1:
 E= -76.4195104502893     Delta-E=       -0.000001796988 Rises=F Damp=F
 DIIS: error= 4.02D-05 at cycle   5 NSaved=   5.
 NSaved= 5 IEnMin= 5 EnMin= -76.4195104502893     IErMin= 5 ErrMin= 4.02D-05
 ErrMax= 4.02D-05  0.00D+00 EMaxC= 1.00D-01 BMatC= 4.14D-08 BMatP= 2.55D-06
 IDIUse=1 WtCom= 1.00D+00 WtEn= 0.00D+00
 Coeff-Com:  0.585D-03-0.609D-03 0.301D-02 0.107D-01 0.986D+00
 Coeff:      0.585D-03-0.609D-03 0.301D-02 0.107D-01 0.986D+00
 Gap=     0.358 Goal=   None    Shift=    0.000
 RMSDP=6.00D-06 MaxDP=7.12D-05 DE=-1.80D-06 OVMax= 4.83D-05

 Initial convergence to 1.0D-05 achieved.  Increase integral accuracy.
 Cycle   6  Pass 1  IDiag  1:
 E= -76.4195112641020     Delta-E=       -0.000000813813 Rises=F Damp=F
 DIIS: error= 3.89D-07 at cycle   1 NSaved=   1.
 NSaved= 1 IEnMin= 1 EnMin= -76.4195112641020     IErMin= 1 ErrMin= 3.89D-07
 ErrMax= 3.89D-07  0.00D+00 EMaxC= 1.00D-01 BMatC= 3.97D-12 BMatP= 3.97D-12
 IDIUse=1 WtCom= 1.00D+00 WtEn= 0.00D+00
 Coeff-Com:  0.100D+01
 Coeff:      0.100D+01
 Gap=     0.358 Goal=   None    Shift=    0.000
 RMSDP=6.00D-06 MaxDP=7.12D-05 DE=-8.14D-07 OVMax= 9.66D-07

 Cycle   7  Pass 1  IDiag  1:
 E= -76.4195112641069     Delta-E=       -0.000000000005 Rises=F Damp=F
 DIIS: error= 1.37D-07 at cycle   2 NSaved=   2.
 NSaved= 2 IEnMin= 2 EnMin= -76.4195112641069     IErMin= 2 ErrMin= 1.37D-07
 ErrMax= 1.37D-07  0.00D+00 EMaxC= 1.00D-01 BMatC= 5.04D-13 BMatP= 3.97D-12
 IDIUse=1 WtCom= 1.00D+00 WtEn= 0.00D+00
 Coeff-Com:  0.171D+00 0.829D+00
 Coeff:      0.171D+00 0.829D+00
 Gap=     0.358 Goal=   None    Shift=    0.000
 RMSDP=3.58D-08 MaxDP=3.00D-07 DE=-4.95D-12 OVMax= 3.49D-07

 Cycle   8  Pass 1  IDiag  1:
 E= -76.4195112641070     Delta-E=       -0.000000000000 Rises=F Damp=F
 DIIS: error= 1.22D-07 at cycle   3 NSaved=   3.
 NSaved= 3 IEnMin= 3 EnMin= -76.4195112641070     IErMin= 3 ErrMin= 1.22D-07
 ErrMax= 1.22D-07  0.00D+00 EMaxC= 1.00D-01 BMatC= 4.49D-13 BMatP= 5.04D-13
 IDIUse=1 WtCom= 1.00D+00 WtEn= 0.00D+00
 Coeff-Com: -0.338D-01 0.483D+00 0.551D+00
 Coeff:     -0.338D-01 0.483D+00 0.551D+00
 Gap=     0.358 Goal=   None    Shift=    0.000
 RMSDP=1.68D-08 MaxDP=1.60D-07 DE=-1.14D-13 OVMax= 1.49D-07

 Cycle   9  Pass 1  IDiag  1:
 E= -76.4195112641073     Delta-E=       -0.000000000000 Rises=F Damp=F
 DIIS: error= 9.98D-09 at cycle   4 NSaved=   4.
 NSaved= 4 IEnMin= 4 EnMin= -76.4195112641073     IErMin= 4 ErrMin= 9.98D-09
 ErrMax= 9.98D-09  0.00D+00 EMaxC= 1.00D-01 BMatC= 2.52D-15 BMatP= 4.49D-13
 IDIUse=1 WtCom= 1.00D+00 WtEn= 0.00D+00
 Coeff-Com: -0.126D-01 0.102D+00 0.155D+00 0.755D+00
 Coeff:     -0.126D-01 0.102D+00 0.155D+00 0.755D+00
 Gap=     0.358 Goal=   None    Shift=    0.000
 RMSDP=1.39D-09 MaxDP=1.58D-08 DE=-2.84D-13 OVMax= 1.46D-08

 SCF Done:  E(RB3LYP) =  -76.4195112641     A.U. after    9 cycles
            NFock=  9  Conv=0.14D-08     -V/T= 2.0077
 KE= 7.583276298780D+01 PE=-1.989440442751D+02 EE= 3.753516768160D+01
 Leave Link  502 at Thu Oct 12 10:24:09 2023, MaxMem=   104857600 cpu:               1.6 elap:               0.8
 (Enter /opt/g16/l601.exe)
 Copying SCF densities to generalized density rwf, IOpCl= 0 IROHF=0.

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Alpha  occ. eigenvalues --  -19.13402  -0.99455  -0.52629  -0.35971  -0.29033
 Alpha virt. eigenvalues --    0.06722   0.15448   0.77839   0.79548   0.88827
 Alpha virt. eigenvalues --    0.89279   1.01923   1.15980   1.51868   1.57874
 Alpha virt. eigenvalues --    1.66283   2.23144   2.24128   2.48000   2.63323
 Alpha virt. eigenvalues --    2.68255   3.03457   3.32839   3.51484   3.72870
          Condensed to atoms (all electrons):
               1          2          3
     1  O    8.042735   0.288393   0.288393
     2  H    0.288393   0.428367  -0.026520
     3  H    0.288393  -0.026520   0.428367
 Mulliken charges:
               1
     1  O   -0.619521
     2  H    0.309761
     3  H    0.309761
 Sum of Mulliken charges =  -0.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  O   -0.000000
 Electronic spatial extent (au):  <R**2>=             19.0010
 Charge=             -0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -0.0000    Y=              0.0000    Z=              1.9536  Tot=              1.9536
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -7.1262   YY=             -4.0552   ZZ=             -6.1125
   XY=              0.0000   XZ=             -0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -1.3616   YY=              1.7094   ZZ=             -0.3478
   XY=              0.0000   XZ=             -0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             -0.0000  YYY=              0.0000  ZZZ=              1.0742  XYY=             -0.0000
  XXY=              0.0000  XXZ=              0.2881  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              1.2049  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -5.1120 YYYY=             -5.9045 ZZZZ=             -6.1400 XXXY=              0.0000
 XXXZ=             -0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=             -0.0000
 ZZZY=              0.0000 XXYY=             -2.1310 XXZZ=             -1.9158 YYZZ=             -1.7462
 XXYZ=              0.0000 YYXZ=             -0.0000 ZZXY=              0.0000
 N-N= 9.156602341551D+00 E-N=-1.989440441008D+02  KE= 7.583276298780D+01
 No NMR shielding tensors so no spin-rotation constants.
 Leave Link  601 at Thu Oct 12 10:24:09 2023, MaxMem=   104857600 cpu:               0.2 elap:               0.1
 (Enter /opt/g16/l9999.exe)
 Unable to Open any file for archive entry.
 1\1\GINC-DESKTOP-1D9QDE1\SP\RB3LYP\6-31G(d,p)\H2O1\JH-LI\12-Oct-2023\0
 \\#p b3lyp em=gd3bj 6-31G** nosymm\\H2O energy task (g16)\\0,1\O,0,0.,
 0.,-0.11085125\H,0,0.,-0.78383672,0.44340501\H,0,0.,0.78383672,0.44340
 501\\Version=ES64L-G16RevC.01\HF=-76.4195113\RMSD=1.393e-09\Dipole=0.,
 0.,0.7685975\Quadrupole=-1.012284,1.2708993,-0.2586153,0.,0.,0.\PG=C02
 V [C2(O1),SGV(H2)]\\@
 The archive entry for this job was punched.


 WHAT THE SLIME CONTAINS, THE GLOWING REVEALS
 BY THE FLAME'S BRIGHT HUE WHEN YOU IGNITE IT.
   --  LEONARD THURNEYSSER, 1531-1596
 DESCRIBING HIS FLAME PHOTOLYSIS ANALYTIC METHOD
 Job cpu time:       0 days  0 hours  0 minutes  3.1 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  1.6 seconds.
 File lengths (MBytes):  RWF=      6 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 16 at Thu Oct 12 10:24:09 2023.
